deepchecks/tabular/suites/default_suites.py

```diff
@@ -47,8 +49,82 @@ def single_dataset_integrity(**kwargs) -> Suite:
     return data_integrity(**kwargs)
 
 
-def data_integrity(**kwargs) -> Suite:
-    """Create a suite that is meant to detect integrity issues within a single dataset."""
+def data_integrity(columns: Union[Hashable, List[Hashable]] = None,
+                   ignore_columns: Union[Hashable, List[Hashable]] = None,
+                   n_top_columns: int = None,
+                   n_samples: int = 1_000_000,
+                   random_state: int = 42,
+                   n_to_show: int = 5,
+                   **kwargs) -> Suite:
+    """Suite for detecting integrity issues within a single dataset.
+
```

Comment:
What do you think about adding here (and for each suite) a note that all of the suite params are transferred directly to each of the checks (thus affecting the relevant checks)?

Reply:
We have the **kwargs parameter that describes that.

Reply:
Yes but I assume most people don't read it as it's in the end + even if they do - they don't necessarily understand that also all of the parameters are also only for the checks,
I though about something like adding one sentence under the "Parameters" title, so that it's highly visible when they consider the parameters, and then it's clearer that *all* of these parameters are actually check-parameters, and none are related to the suite itself. WDYT